browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -405,6 +403,15 @@ views::View* BraveBrowserView::GetWalletButtonAnchorView() {
       ->GetAsAnchorView();
 }
 
+std::map<int, std::vector<ui::Accelerator>>
+BraveBrowserView::GetAcceleratedCommands() {
+  std::map<int, std::vector<ui::Accelerator>> result;
```

Comment:
I guess this result could be mutable in the future, as this PR is in pre-stage for customizing the shortcuts, isn't it? Could you leave a comment about it?